spaceship/lib/spaceship/base.rb

```diff
@@ -59,6 +59,9 @@ def each(&block)
       def to_json(*a)
         h = @hash.dup
         h.delete(:application)
+        if h.key?(:version) && h[:version].kind_of?(Spaceship::Tunes::AppVersion)
```

Comment:
I think maybe the best move would be to copy the `raw_data` in the `complete!` method, remove the version from that copy, and then use that copied instance to pass to `send_app_submission` on the client. Does that make sense? 🐙